src/util/events.js

```diff
@@ -49,12 +56,6 @@ export function getFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      // We do not yet have valid zipcodes to filter against,
-      // give it a pass.
-      if (!zipcodes || !zipCodesLength) {
-        return true;
-      }
-
       const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
```

Comment:
This explodes if you enter an invalid zipcode (e.g. 00000, or a slew of other close-to-valid-but-a-typo ones): `TypeError: Cannot read property 'distanceTo' of undefined`. Need to check that `zipcodes[filters.zipcode] !== undefined` first.